langchain/src/retrievers/amazon_knowledge_bases.ts

```diff
@@ -0,0 +1,6 @@
+import { logVersion010MigrationWarning } from "../util/entrypoint_deprecation.js";
```

Comment:
Let's just export from `@langchain/community`, can you revert all changes in `langchain`?

Reply:
Sure, reverted the langchain changes.